src/presenter/presenter-list.js

```diff
@@ -0,0 +1,45 @@
+import {render} from '../render.js';
+import FilterView, {filtersContainer} from '../view/view-filters.js';
+import SortView, {tripEventsContainer} from '../view/view-sort.js';
+import EditingFormView from '../view/form/view-editing-form.js';
+import PointsContainerView from '../view/view-points-container.js';
+import PointView from '../view/point/view-point.js';
+
+
+export default class ListPresenter {
+  pointsContainerView = new PointsContainerView();
+
+  constructor ({pointsModel, destinationsModel, offersModel}) {
+    this.pointsModel = pointsModel;
+    this.destinationsModel = destinationsModel;
+    this.offersModel = offersModel;
+    this.arrayDestinationsModel = destinationsModel.destinations;
+    this.arrayPointsModel = pointsModel.points;
+  }
+
+  init() {
+
+    render(new FilterView(), filtersContainer);
+    render(new SortView(), tripEventsContainer);
+    render(new EditingFormView(
+      this.pointsModel.getPoint(),
+      this.destinationsModel.getById(this.pointsModel.getPoint().destination),
+      this.offersModel.getByType(this.pointsModel.getPoint().type),
+      this.arrayDestinationsModel),
+    tripEventsContainer);
+
```

Comment:
учитывая что getPoint выдаёт рандомный поинт, у тебя получается что офферы которые ты получаешь вообще не подходят к тому поинту который ты передаёшь, либо убери рандом, и выводи например первый, либо уже пиши что то типа 
```suggestion
   const editFormPoint = this.pointsModel.getPoint()
    render(new EditingFormView(
      editFormPoint,
      this.destinationsModel.getById(editFormPoint.destination),
      this.offersModel.getByType(editFormPoint.type),
      this.arrayDestinationsModel),
    tripEventsContainer);

```